### PR TITLE
multi: bump lnd to version v0.13.0-beta.rc2

### PR DESCRIPTION
**Pull request by guggero**
Depends on https://github.com/lightninglabs/faraday/pull/119
Depends on https://github.com/lightninglabs/loop/pull/388
Depends on https://github.com/lightninglabs/pool/pull/256

Bumps all integrated daemon versions to their latest tagged release that supports compile time compatibility with `lnd v0.13.0-beta`.

### CODE REVIEW

**Comment by guggero** [comment]
The RPC check can only be fixed once the dependent PRs are in and we have tagged them!

**Comment by Roasbeef** [comment]
Thanks for putting this together! Running this now on one of my mainnet nodes.

**Comment by Roasbeef** [comment]
Ran into this error when attempting to update: 
```
goroutine 1 [running]:
github.com/lightningnetwork/lnd/build.(*RotatingLogWriter).SetLogLevels(0x0, 0x1810d3a, 0x4)
        /root/go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc2/build/logrotator.go:150 +0x37
github.com/lightningnetwork/lnd/build.ParseAndSetDebugLevels(0x1810d3a, 0x4, 0x28c9458, 0x0, 0x27, 0x13d86d5)
        /root/go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc2/build/log.go:142 +0x703
github.com/lightninglabs/faraday.ValidateConfig(0x32d6e80, 0x0, 0x0)
        /root/go/pkg/mod/github.com/lightninglabs/faraday@v0.2.3-alpha.0.20210517133456-824df2db313f/config.go:231 +0x499
github.com/lightninglabs/lightning-terminal.loadAndValidateConfig(0xc000138540, 0xc0003c60c0, 0xc0003c6120, 0xc0003c6180, 0x0, 0x0, 0x0)
        /root/go/src/github.com/lightninglabs/lightning-terminal/config.go:381 +0x596
github.com/lightninglabs/lightning-terminal.(*LightningTerminal).Run(0xc000414600, 0x0, 0x0)
        /root/go/src/github.com/lightninglabs/lightning-terminal/terminal.go:147 +0x1b1
main.main()
        /root/go/src/github.com/lightninglabs/lightning-terminal/cmd/litd/main.go:14 +0x89
```

**Comment by guggero** [comment]
Thanks, I fixed the error in the logging.

I realized we have another thing that we're running into with this. We need a way to detect whether `lnd` is 0.13 or before and adjust the mechanism that waits for `lnd` to unlock.
So we basically need to fix the compatibility/unlock check in `lndclient` to make sure we support both 0.13 and 0.12 remote nodes.

**Comment by Roasbeef** [comment]
> Thanks, I fixed the error in the logging.

Is this commit pushed? Things still panic for me.

**Comment by jamaljsr** [comment]
Also got the same error as @Roasbeef after updating the `go.mod` to point to the latest commits of the loop and pool PRs.

**Comment by guggero** [comment]
No, I didn't push yet as it would've failed in a later place anyway.
The `lndclient` problem is now fixed and nothing should crash anymore. Sorry for not testing it better before asking for review!

File: terminal.go [hunk 1]
```diff
@@ -912,10 +910,7 @@ func (g *LightningTerminal) showStartupInfo() error {
 		ctx := context.Background()
 		res, err := basicClient.GetInfo(ctx, &lnrpc.GetInfoRequest{})
 		if err != nil {
-			s, ok := status.FromError(err)
-			if !ok || s.Code() != codes.Unimplemented {
-				// Some other error that we didn't expect at
-				// this moment.
+			if !lndclient.IsUnlockError(err) {
```
Comment (Roasbeef) [inline terminal.go:913]: 👍

**Comment by Roasbeef** [comment]
Getting this error when trying to build: 
```
TypeScript error in /root/go/src/github.com/lightninglabs/lightning-terminal/app/src/util/tests/sampleData.ts(74,14):
Property 'accountBalanceMap' is missing in type '{ totalBalance: number; confirmedBalance: number; unconfirmedBalance: number; }' but required in type 'AsObject'.  TS2741

    72 | };
    73 |
  > 74 | export const lndWalletBalance: LND.WalletBalanceResponse.AsObject = {
       |              ^
    75 |   totalBalance: 84992363,
    76 |   confirmedBalance: 84992363,
    77 |   unconfirmedBalance: 0,


error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Perhaps it's my node version?

**Review by Roasbeef — COMMENTED** [review]
Changes look good, but I still seem to be having some build issues on my end, might just be related to my environmen though.

**Comment by jamaljsr** [comment]
> Getting this error when trying to build:
> 
> ```
> TypeScript error in /root/go/src/github.com/lightninglabs/lightning-terminal/app/src/util/tests/sampleData.ts(74,14):
> Property 'accountBalanceMap' is missing in type '{ totalBalance: number; confirmedBalance: number; unconfirmedBalance: number; }' but required in type 'AsObject'.  TS2741
> 
>     72 | };
>     73 |
>   > 74 | export const lndWalletBalance: LND.WalletBalanceResponse.AsObject = {
>        |              ^
>     75 |   totalBalance: 84992363,
>     76 |   confirmedBalance: 84992363,
>     77 |   unconfirmedBalance: 0,
> 
> 
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> ```
> 
> Perhaps it's my node version?

I fixed this in https://github.com/lightninglabs/lightning-terminal/pull/226/commits/2293f16e5f6eda58c93384915777eec1677815cd

**Review by Roasbeef — APPROVED** [review]
LGTM 🦖

Works for me after jamal's last commit!

**Comment by guggero** [comment]
Thank you @jamaljsr for the fix! Should've checked the JS/TS compilation after updating the protos :man_facepalming:

**Review by jamaljsr — APPROVED** [review]
LGTM 🥇 

Tested on regtest and testnet 👍

**Pull request merged**